includes/discovery/sensors/current/cimc.inc.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$pwr_board = snmpwalk_array_num($device, '.1.3.6.1.4.1.9.9.719.1.9.14');
+
+/*
+* False == OID not found - this is not an error.
+* null  == timeout or something else that caused an error.
+*/
+if (is_null($pwr_board)) {
```

Comment:
We don't need this or the else in. Just check the `$pwr_board` is not empty and then run your code.

Reply:
Same for the other files.